src/Microsoft.DotNet.ProjectJsonMigration/MigrationNuGetFrameworkExtensions.cs

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using NuGet.Frameworks;
+
+namespace Microsoft.DotNet.ProjectJsonMigration
+{
+    public static class MigrationNuGetFrameworkExtensions
+    {
+        public static string GetMSBuildCondition(this NuGetFramework framework)
+        {
+            return $" '$(TargetFrameworkIdentifier),Version=$(TargetFrameworkVersion)' == '{framework.DotNetFrameworkName}' ";
```

Comment:
I think you can just say `'$(TargetFramework)' == '{framework.ShortName}'`

/cc @nguerrera - to confirm if this is true.